instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java

```diff
@@ -277,7 +279,8 @@ public Searcher forwardBackendSearchResult(@NonNull JSONObject response) {
 
         updateListeners(response, false);
         updateFacetStats(response);
-        EventBus.getDefault().post(new ResultEvent(response, query, REQUEST_UNKNOWN));
+        SearchResults results = new SearchResults(response, hierarchicalFacets);
```

Comment:
Might as well inline `results` in the next line

Reply:
Put it inline will make the next line too long.